userbenchmark/cpu/run_config.py

```diff
@@ -0,0 +1,106 @@
+
+"""
+Run PyTorch cpu benchmarking.
+"""
+import argparse
+import os
+import sys
+import numpy
+
+from typing import List, Tuple, Dict, Optional
+from pathlib import Path
+
+REPO_DIR = str(Path(__file__).parent.parent.parent.resolve())
+sys.path.append(REPO_DIR)
```

Comment:
If you are using `with add_path` later, why set it globally?
```suggestion
```

Reply:
Due to we need use the `add_path` class in `utils`. But I have copy the `add_path` into `cpu_utils.py` also, we can remove the this line.